fendermint/actors/gas_market/src/lib.rs

```diff
@@ -8,28 +8,42 @@ use fil_actors_runtime::{actor_dispatch, ActorError};
 use fvm_ipld_encoding::tuple::*;
 use fvm_shared::econ::TokenAmount;
 use fvm_shared::METHOD_CONSTRUCTOR;
-use lazy_static::lazy_static;
 use num_derive::FromPrimitive;
 use std::cmp::Ordering;
 
 #[cfg(feature = "fil-actor")]
 fil_actors_runtime::wasm_trampoline!(EIP1559GasMarketActor);
 
-/// Base fee max change denominator as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
-const BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 8;
-/// Elasticity multiplier as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
-const ELASTICITY_MULTIPLIER: u64 = 2;
-lazy_static! {
-    /// Initial base fee as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
-    static ref INITIAL_BASE_FEE: TokenAmount = TokenAmount::from_atto(1_000_000_000);
-    static ref MINIMAL_BASE_FEE: TokenAmount = TokenAmount::from_atto(100);
-}
 pub const IPC_GAS_MARKET_ACTOR_NAME: &str = "gas_market";
 pub type Gas = u64;
-pub type GasMarketReading = EIP1559GasState;
+
+/// Constant params used by EIP1559
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone)]
+pub struct EIP1559Constants {
+    /// The minimal base fee when gas utilization is low
+    minimal_base_fee: TokenAmount,
+    /// Elasticity multiplier as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
+    elasticity_multiplier: u64,
+    /// Base fee max change denominator as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
+    base_fee_max_change_denominator: u64,
+}
 
 #[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone)]
 pub struct EIP1559GasState {
+    block_gas_limit: Gas,
```

Comment:
I don't get why the gas limit hasn't been moved into the constants. With this latest change, we can now configure everything _but_ the block gas limit. Is that what you intended? If so, why?